libraries/theme/src/main/kotlin/io/element/android/libraries/theme/compound/generated/TypographyTokens.kt

```diff
@@ -35,110 +23,142 @@ object TypographyTokens {
         lineHeight = 22.sp,
         fontSize = 16.sp,
         letterSpacing = 0.015629999999999998.em,
+        platformStyle = PlatformTextStyle(includeFontPadding = false),
```

Comment:
You mean also `includeFontPadding = false` is not a default?

Reply:
It is false by default now, but `LineHeightStyle` depends on it being false to work fine, so I think it's better if we're explicit.

Reply:
I left it there because I thought we might need to add it for custom styles, but:

1. There should be no custom styles.
2. If there are, they should be derived from our Compound ones and they would already have the right line height style.

So there's probably no need for it anymore.